package.json

```diff
@@ -11,10 +11,12 @@
     "dist"
   ],
   "scripts": {
+    "commit": "northbrook commit",
```

Comment:
How about we pick *either* the name `nb` or `northbrook` and use it in all the scripts, rather than mixing--it's `northbrook` everywhere except the `"build"` script atm.  Any preference?